### ISSUE TITLE
Change Swatch Size

### ISSUE THREAD

**Issue by Visnicio** [Naeemur/react-native-wheel-color-picker#34]
I've been reading the docs but couldn't find any solutions so increase/decrease the swatches size, but I found in the source code that this is made with just a simple style.

By now i'm just a begginer in react native and cant make a fork to code a solution for that, could you please make a parameter for that?

**Comment by Wissam-Yelhow** [comment]
+1

**Comment by jigarsolanki0212** [comment]
Yes, I'm also trying to reduce the Swatches size but couldn't find any props. please make a prop for it.
@Naeemur

**Comment by dcorbin** [comment]
Related, but not the same.  I would like to see an option for the swatches being in multiple rows.  Maybe specify swatchRowCount={2}.  For me, one row will be common standard colors, and the other row will be the last 8-10 selected colors.

**Comment by Visnicio** [comment]
Hey guys, this repo does not seem to be maintained, the developer made some updates but no progress on this issue.
I had a workaround for the swatch size I can try to create a PR to solve this problem and create the feature
